server.js

```diff
@@ -0,0 +1,24 @@
+require('./src/config');
+const express = require('express');
+const argv = require('minimist');
+const path = require('path');
+const healthcheck = require('express-healthcheck');
+
+const args = argv(process.argv.slice(2));
+const HTTP_PORT = process.env.PORT || args.port || 3000;
+const server = express();
+
+
+// ********************* MIDDLEWARE **************************
+
+server.use('/health', healthcheck());
```

Comment:
This would be pretty easy to write a test for, see https://github.com/goldbergyoni/nodebestpractices/blob/master/sections/projectstructre/separateexpress.md#example-test-your-api-in-process-using-supertest-popular-testing-package